backend/app/Services/Erp/MaterialLotImportService.php

```
@@ -0,0 +1,202 @@
+
+    /**
+     * Re-derive each touched material's warehouse total from its lot rows, so the
+     * bulk row keeps meaning "everything of this material in this warehouse".
+     *
+     * @param  array<string, array{0: int, 1: int}>  $pairs
+     */
+    private function reconcileWarehouseTotals(array $pairs): void
+    {
+        foreach ($pairs as [$warehouseId, $materialId]) {
+            DB::transaction(function () use ($warehouseId, $materialId) {
+                $total = (float) WarehouseStock::query()
```

PHPStan (2.2.6)

[error] Call to an undefined static method App\Models\WarehouseStock::query().


(staticMethod.notFound)